### DISCUSSION TITLE
Mermaid extension: Manually change size of diagram

### DISCUSSION THREAD

**Discussion by PhilippAumann** [outline/outline#7020] [Ideas]
The mermaid extension is very helpful, however when creating large flowcharts, the created diagram is quite small such that the text is barely readable and I couldn't find a way to resize the output of a mermaid diagram within Outline. Is this somehow possible?

**Comment by tommoor** [comment]
It's not possible currently, but it could be built fairly easily as the pattern already exists for resizing images and videos.
